### ISSUE TITLE
how do you get metrics like speficicity and sensitivity with masks?

### ISSUE THREAD

**Issue by AmanitaCreaciones** [facebookresearch/Detectron#728]
Hi,

I've been experimenting with Mask RCNN and I would like to know with other metrics you use to see how good it performs more than just loss.

when classificating with bounding boxes I could somehow calculate speficicity and sensitivity depending on the distance between the predicted bounding box and the real object to locate, but with masks I'm not sure how to do it.

Any advice?

**Comment by ir413** [comment]
Hi @AmanitaCreaciones, in addition to the training loss we look at COCO evaluation metrics on the validation set (see [here](http://cocodataset.org/#detection-eval) for more info). You should be able to compute the specificity and sensitivity metrics by performing IoU based matching.
